modules/Core/pages/panel/discord.php

```diff
@@ -0,0 +1,136 @@
+<?php
+/*
+ *	Made by Samerton
```

Comment:
Is it? :P

Reply:
👀 I thought that meant NamelessMC as a whole, not the page

Reply:
Not sure to be honest, I never understood the point of header comments anyway